docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,92 @@
+# Quick start
+
+## What you'll do
+
+- Setup a GitHub repository
+- Activate the GitHub APIs related to self-hosted runner
+- Deploy the [GitHub runner charm](https://charmhub.io/github-runner)
+- Ensure GitHub repository setting are secure
+- Run a simple workflow on the self-hosted runner
+
+## Requirements
+
+- GitHub Account.
+- Juju 3 installed.
+- Juju controller on OpenStack, and a juju model.
+
+For more information about how to install and use Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
+
+## Steps
+
+### Create GitHub repository
+
+The GitHub self-hosted runner spawned by the charm needs to connect to a GitHub repository or organization. GitHub repositories are used as it is simpler to manage.
+
+To create a GitHub repository, log in to [GitHub](https://github.com) with your GitHub Account and follow the instruction [here](https://docs.github.com/en/get-started/quickstart/create-a-repo#create-a-repository). For this tutorial, create a public repository as [GitHub branch protection rules is only available in public repository with GitHub Free tier](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule).
+
+### Activate GitHub APIs related to self-hosted runner
+
+***This must be done for GitHub runner charm to function correctly.***
+
+The GitHub runner charm uses GitHub APIs related to self-hosted runners. Some of the APIs will only be functional after a self-hosted runner registration token is requested for the repository for the first time. If a registration token is requested for the repository before then this can be skipped.
+
+The registration token can be requested by calling the [GitHub API to request a self-hosted runner registration token](https://docs.github.com/en/rest/actions/self-hosted-runners?apiVersion=2022-11-28#create-a-registration-token-for-a-repository). Alternatively, a registration token can be requested by visiting the "New self-hosted runner" page for the repository (`https://github.com/{OWNER}/{REPO}/settings/actions/runners/new`). This can be done by following the instruction to the 4th step provided [here](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners#adding-a-self-hosted-runner-to-a-repository).
```

Comment:
s/GitHub API to request a self-hosted runner registration token/GitHub API
s/Alternatively, a registration token can/Alternatively, it can also